src/EbayAccessTests/EbayBaseResponseTest.cs

```
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using EbayAccess.Models;
+using EbayAccess.Models.BaseResponse;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace EbayAccessTests
+{
+	[ TestFixture ]
+	public class EbayBaseResponseTest
+	{
+		private readonly List< ResponseError > _errorsForIgnoring = new List< ResponseError >
+		{
+			EbayErrors.EbayPixelSizeError,
+			EbayErrors.LvisBlockedError,
+			EbayErrors.UnsupportedListingType,
+			EbayErrors.ReplaceableValue,
+			EbayErrors.MpnHasAnInvalidValue,
+			EbayErrors.DuplicateListingPolicy,
+			EbayErrors.OperationIsNotAllowedForInventoryItems,
+			EbayErrors.InvalidMultiSkuItemId,
+			EbayErrors.AuctionEnded
+		};
+
+		[ TestCase( EbayErrors.EbayPixelSizeErrorCode ) ]
+		[ TestCase( EbayErrors.LvisBlockedErrorCode ) ]
+		[ TestCase( EbayErrors.UnsupportedListingTypeErrorCode ) ]
+		[ TestCase( EbayErrors.ReplaceableValueErrorCode ) ]
+		[ TestCase( EbayErrors.MpnHasAnInvalidValueErrorCode ) ]
+		[ TestCase( EbayErrors.DuplicateListingPolicyErrorCode ) ]
+		[ TestCase( EbayErrors.OperationIsNotAllowedForInventoryItemsErrorCode ) ]
+		[ TestCase( EbayErrors.InvalidMultiSkuItemIdErrorCode ) ]
+		[ TestCase( EbayErrors.AuctionEndedErrorCode ) ]
+		public void SkipErrorsAndDo_ReturnResponseWithoutIgnoringErrors( string errorCode )
```

I'm not sure I understand what code this test checks. Looks like it tests `SkipErrorsAndDo` logic, but you changed `ReviseFixedPriceItemsAsync` part. Maybe that method could be refactored a bit, or something, to be able to test, but now this tests gives us nothing in terms of testing our changes.

---

Or you can describe why these changes could not be tested without big refactoring, if it is. That's an option too.

---

I don't want refactor anything here even on pain of death. But I reproduced problem and added tests like you wanted. I hope it is enough.

---

New test looks good, it would reproduce the problem before the changes. 

This one looks a bit srange for me, not sure what's the puspose in having all these error codes listed. If we want to have test for `EbayBaseResponseTest`, I think it make sense to leave only one error in `_errorsForIgnoring`.
And please name it SkipErrorsAndDo_ReturnResponseWithoutIgnor**ed**Errors, otherwise it means that we should not ignore errors.
And potentially, you can add a test for throwing when error is not ignored - `SkipErrorsAndDo_Throws_WhenErrorIsNotIgnored`.
But this is optional. In fact you did not need this method tests at all for this bug. But if you wrote some to improve testability here - let's make to follow our testing practices - clear, atomic, testing one thing at a time, etc.

---

@andrey-kozlov-skuvault Fixed. That's ok?